clients/devices.js

```diff
@@ -3,13 +3,19 @@
 module.exports = {
   name: 'Is the devices list settings section visible',
   startDate: '2015-01-01',
-  subjectAttributes: ['forceDeviceList', 'isMetricsEnabledValue'],
+  subjectAttributes: ['forceDeviceList', 'uid'],
   independentVariables: ['deviceListVisible'],
   eligibilityFunction: function (subject) {
+    var sampleRate = 0.1;
+
     if (subject) {
-      if (subject.isMetricsEnabledValue || subject.forceDeviceList) {
+      if (subject.forceDeviceList) {
         return true;
       }
+
+      if (subject.uid) {
+        return !! (this.bernoulliTrial(sampleRate, subject.uid));
```

Comment:
this is based on https://github.com/mozilla/fxa-content-experiments/blob/dev/metrics/is-sampled-user.js#L9